.golangci.yml

```diff
@@ -0,0 +1,10 @@
+run:
```

Comment:
What would happen if we used the same config file as the [app one](https://github.com/docker/app/blob/master/.golangci.yml)?

Reply:
I just didn't noticed we already migrated app.
Should apply to other docker/* repos